package.json

```
@@ -25,6 +26,7 @@
     "gatsby-transformer-remark": "^6.13.0",
     "gatsby-transformer-sharp": "^5.13.0",
     "lottie-react": "^2.4.0",
+    "@lottie-animation-community/lottie-specs": "^0.3.1",
```

duplicated dependency

```suggestion
    "@lottie-animation-community/lottie-specs": "^0.3.1",
```